### PR TITLE
KTLN-770 Deserialize Json Object Property to a String

### PR DESCRIPTION
**Pull request by nappy29**
Added unit tests

### CODE REVIEW

File: core-kotlin-modules/core-kotlin-strings-4/src/test/kotlin/com/baeldung/deserializestringobject/DeserializeStringObjectUnitTest.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        assertEquals("John", name)
+        assertEquals("New York", city)
+        assertEquals("30", age)
+        assertEquals("{\"name\":\"John\",\"age\":30,\"city\":\"New York\"}", jsonObject.toString())
+    }
+
+    @Test
+    fun `deserialize Json object to string using Kotlinx serialization`(){
+        val json = Json { }
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-strings-4/src/test/kotlin/com/baeldung/deserializestringobject/DeserializeStringObjectUnitTest.kt:32]: This line doesn't match what is in the article at the moment.

File: core-kotlin-modules/core-kotlin-strings-4/src/test/kotlin/com/baeldung/deserializestringobject/DeserializeStringObjectUnitTest.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        assertEquals("New York", city)
+        assertEquals("30", age)
+        assertEquals("{\"name\":\"John\",\"age\":30,\"city\":\"New York\"}", jsonObject.toString())
+    }
+
+    @Test
+    fun `deserialize Json object to string using Jackson library`(){
+        val objectMapper = jacksonObjectMapper()
+        val jsonString = "{\"name\":\"John\", \"age\":30, \"city\":\"New York\"}"
+        val jsonObject = objectMapper.readValue(jsonString, JsonNode::class.java)
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-strings-4/src/test/kotlin/com/baeldung/deserializestringobject/DeserializeStringObjectUnitTest.kt:50]: let's use the `readValue` extension function for this.

```
objectMapper.readValue<JsonNode>(jsonString)
```

File: core-kotlin-modules/core-kotlin-strings-4/pom.xml [hunk 3]
```diff
@@ -14,6 +14,25 @@
...
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.10.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-serialization-json-jvm</artifactId>
+            <version>1.6.2</version>
+            <scope>runtime</scope>
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-strings-4/pom.xml:27]: It feels like this should be declared as the default scope instead.

File: core-kotlin-modules/core-kotlin-strings-4/pom.xml [hunk 4]
```diff
@@ -14,6 +14,7 @@
     </parent>
 
     <dependencies>
+
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-strings-4/pom.xml:17]: No reason to change this file. Please revert.

**Pull request merged**